tools/mo/openvino/tools/mo/ops/Complex.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from openvino.tools.mo.graph.graph import Node, Graph
+from openvino.tools.mo.ops.op import Op
+
+
+class Complex(Op):
```

Comment:
I discussed this topic with @pavel-esir. He said that we needn't `reverse_infer` and `type_infer` for this operation, because resulting IR doesn't contain such operation.